src/gui/cli/ServerData.hpp

```diff
@@ -11,7 +11,7 @@ namespace Zappy::GUI {
     struct ServerData
     {
             std::pair<unsigned int, unsigned int> _mapSize;
-            std::vector<std::vector<std::string>> _mapTiles;
+            std::vector<std::vector<int>> _mapTiles;
```

Comment:
please create a Tile class that contains 7 int attributes: food and stones quantity